uni/lib/view/locations/locations.dart

```diff
@@ -52,29 +53,70 @@ class LocationsPageView extends StatelessWidget {
   final List<LocationGroup> locations;
   final RequestStatus status;
 
+  @override
+  LocationsPageViewState createState() => LocationsPageViewState();
+}
+
+class LocationsPageViewState extends State<LocationsPageView> {
+  static GlobalKey<FormState> searchFormKey = GlobalKey<FormState>();
+  static String searchTerms = '';
```

Comment:
Does this need to be static? since you are using setState?

Reply:
It does since the entire page rerenders when the keyboard opens and so does the search field, effectively hiding the keyboard again. This is done to avoid that